packages/ui/src/components/pages/Accounts/AccountAddress.tsx

```diff
@@ -11,21 +12,21 @@ interface AccountAddressProps extends Props {
 }
 
 const AccountAddress: FC<AccountAddressProps> = ({ className, address, name }) => {
-  const { width } = useWindowSize();
-  const showShortAddress = width < 500;
+  const showShortAddress = useMediaQuery('(max-width: 600px)');
   const getDisplayAddress = () => {
     return showShortAddress ? shortenAddress(address) : address;
   };
 
   const [displayAddress, setDisplayAddress] = useState(getDisplayAddress());
 
-  useEffect(() => {
+  useUpdateEffect(() => {
     setDisplayAddress(getDisplayAddress());
-  }, [width, address]);
+  }, [showShortAddress, address]);
 
   return (
-    <CopyAddressTooltip address={address} name={name}>
-      <div className={className}>{displayAddress || <span>&nbsp;</span>}</div>
+    <CopyAddressTooltip className='flex gap-1 items-center' address={address} name={name}>
```

Comment:
We should fix the position for this tooltip when the address is in short format
<img width="576" alt="Screenshot 2023-05-07 at 15 08 39" src="https://user-images.githubusercontent.com/6867026/236665780-cd3b187a-70f1-4527-899c-deeea648a8a1.png">